jvm-streaming/src/Language/Java/Streaming.hs

```diff
@@ -40,23 +40,21 @@ import Streaming (Stream, Of)
 import qualified Streaming.Prelude as Streaming
 import System.Mem.Weak (addFinalizer)
 
-isPoppableStream :: IORef (Stream (Of a) IO ()) -> IO Word8
+isPoppableStream :: IORef (Maybe (a, Stream (Of a) IO ())) -> IO Word8
 isPoppableStream ref = do
-    stream <- readIORef ref
-    Streaming.uncons stream >>= \case
+    readIORef ref >>= \case
       Nothing -> return $ fromIntegral $ fromEnum False
-      Just _ -> return $ fromIntegral $ fromEnum True
+      Just _  -> return $ fromIntegral $ fromEnum True
```

Comment:
no whitespace alignment. Just clutters diffs and increases likelihood of conflicts.